test/unit/util.js

```diff
@@ -9,7 +9,7 @@ test('token balance stringify 1', function (t) {
 
   const result = util.stringifyBalance(balance, decimals)
 
-  t.equal(result, '1.269', 'Creates correct balance.')
+  t.equal(result, '1.26973', 'Creates correct balance.')
```

Comment:
Given my suggestion about continuing to default to 3, it would be good to add an extra test, so that we test the default case and the case where a non-default (e.g. 5) is passed in

Reply:
@danjm updated